test/e2e/e2e.go

```diff
@@ -188,14 +189,48 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 
 })
 
+var lock sync.Mutex
```

Comment:
`lock` seems like a very generic name, given how much we have in the `e2e` package.